index.html

```diff
@@ -1,11 +1,582 @@
+      <div class="container" data-aos="fade-up">
+        <div class="section-header">
+          <h2>Timeline</h2>
+          <p>Here is our timeline</p>
+        </div>
+
+        <ul class="nav nav-tabs" role="tablist" data-aos="fade-up" data-aos-delay="100">
+          <li class="nav-item">
+            <a class="nav-link active" href="#day-1" role="tab" data-toggle="tab">November 28th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-2" role="tab" data-toggle="tab">December 7th</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#day-3" role="tab" data-toggle="tab">December 11th</a>
+          </li>
+        </ul>
+
+        <div class="tab-content row justify-content-center" data-aos="fade-up" data-aos-delay="200">
+
+          <!-- Schdule Day 1 -->
+          <div role="tabpanel" class="col-lg-9 tab-pane fade show active" id="day-1">
+            <br>
+            <h1 class="sub-heading"><b>Essay Abstract Due!</b><br></h1>
+            <h3 class="sub-headin">Please submit your essay abstract by 11:59 PM EST on Saturday, <b>November 28th</b>. A submission form will be available soon.</h3>
+          </div>
+          <!-- End Schdule Day 1 -->
+
+          <!-- Schedule Day 2 --> 
+          <div role="tabpanel" class="col-lg-9  tab-pane fade" id="day-2">
+            <br>
+            <h1 class="sub-heading"><b>Complete Essay Due!</b><br></h1>
+            <h3 class="sub-headin">Please submit your <b>complete</b> essay by 11:59 PM EST on Monday, <b>December 7th</b>. Essays will be reviewed by the committee and published on the website by <b>December 9th</b>. A submission form will be available soon.</h3>
+          </div>
+          <!-- End Schdule Day 2 -->
+
+          <!-- Schdule Day 3 -->
+          <div role="tabpanel" class="col-lg-9  tab-pane fade" id="day-3">
+            <br>
+            <h1 class="sub-heading"><b>MIT Pathfinders Event!</b><br></h1>
+            <h3 class="sub-headin">Join us for our <b>live</b> MIT Pathfinders Event with various vaccine experts and professors. We will go over submitted essays and host a QaA with the experts!</h3>
+          </div>
+          <!-- End Schdule Day 3 -->
+        </div>
+      </div>
+    </section> <!-- End Schedule Section -->
+    <!-- ======= Register Section ======= -->
+    <section id="register">
+      <div class="container" data-aos="zoom-in">
+        <div class="section-header-inv">
+          <h2>Register</h2>
+          <!-- <p>Join our Slack!</p> -->
+          <!-- <p>Please read our <a style="color:white; text-decoration: underline;" href="rules.html">Eligibility & Rules</a> before registering for the competition.</p> -->
+        </div>
+        <center>
+          <p style="width: 80%">
+            To register, you must be at least 13 years old <b>and</b> in high school. Register now through the Google Forms, or click the button.
```

Comment:
Can you clarify here that this is registering to attend the event, but not registerign to submit an essay